README.md

```diff
@@ -20,7 +20,7 @@
 
 ## Overview
 
-The sqlserver module installs and manages Microsoft SQL Server 2012, 2014, 2016 and 2017 on Windows systems.
+The sqlserver module installs and manages Microsoft SQL Server 2012, 2014, 2016, 2017, 2019 on Windows systems.
```

Comment:
Do we need to make the corresponding changes in `README_ja_JP.md` or is that handled by translators @ehom ?